packages/manager/apps/pci-private-network/src/components/global-regions/GlobalRegionsDatagrid.tsx

```diff
@@ -0,0 +1,246 @@
+        <td>{renderText(subnetDetail.cidr)}</td>
+        <td>
+          {subnetDetail.gatewayName && (
+            <OsdsLink
+              href={`${projectUrl}/gateway`}
+              color={ODS_THEME_COLOR_INTENT.primary}
+              className="block"
+            >
+              {subnetDetail.gatewayName}
+            </OsdsLink>
+          )}
+          {subnetDetail.gatewayIp &&
+            subnetDetail.dhcpEnabled &&
+            renderText(subnetDetail.gatewayIp)}
+        </td>
+        <td>
+          {renderText(
+            subnetDetail.dhcpEnabled
+              ? renderChip(
+                t('pci_projects_project_network_private_dhcp_active'),
+                ODS_THEME_COLOR_INTENT.success,
+              )
+              : renderChip(
+                t('pci_projects_project_network_private_dhcp_disabled'),
+                ODS_THEME_COLOR_INTENT.warning,
+              ),
+          )}
+        </td>
+        <td>{renderText(subnetDetail.allocatedIp)}</td>
+        <td className="text-right pr-6">
+          {subnetDetail.gatewayIp &&
+            renderTooltipButton(
+              ODS_ICON_NAME.SETTINGS,
+              t('pci_projects_project_network_private_assign_gateway'),
+            )}
+          {renderTooltipButton(
+            ODS_ICON_NAME.BIN,
+            t('pci_projects_project_network_private_delete'),
+          )}
+        </td>
+      </tr>
+    );
+
+    return (
+      <>
+        {network.subnets.map((subnetDetail: TSubnet, index: number) =>
+          renderSubnetRow(subnetDetail, index),
+        )}
+      </>
+    );
+  };
+
+  const renderHeader = (): JSX.Element => {
+    const renderText = (text?: string) => (
+      <OsdsText
+        size={ODS_THEME_TYPOGRAPHY_SIZE._500}
+        hue={ODS_TEXT_COLOR_HUE._600}
+        className="text-center h-11 whitespace-nowrap"
+      >
+        {text}
+      </OsdsText>
+    );
+
+    return (
+      <tr>
+        <th className="p-4">
+          {renderText(t('pci_projects_project_network_private_vlan_id'))}
+        </th>
+        <th className="p-4">
+          {renderText(t('pci_projects_project_network_private_name'))}
+        </th>
+        <th className="p-4">
+          {renderText(t('pci_projects_project_network_private_region'))}
+        </th>
+        <th className="p-4">{renderText('CIDR')}</th>
+        <th className="p-4">
+          {renderText(t('pci_projects_project_network_private_gateway'))}
+        </th>
+        <th className="p-4">{renderText('DHCP')}</th>
+        <th className="p-4">
+          {renderText(t('pci_projects_project_network_private_ip_allocation'))}
+        </th>
+        <th className="w-[8rem]">{renderText()}</th>
+      </tr>
+    );
+  };
+
+  return (
+    <div className="container contents overflow-x-auto">
+      <table className="w-full mb-2 border-collapse">
+        <thead>{renderHeader()}</thead>
+        <tbody>{items.map((network) => renderBodyRow(network))}</tbody>
```

Comment:
```suggestion
        <thead><Header /></thead>
        <tbody>{items.map((network) => <BodyRow {...network} />)}</tbody>
```

would be more natural for example